R/forward_geocoding.R

```diff
@@ -69,6 +69,38 @@ oc_forward <-
     oc_parse(res)
   }
 
+#' @export
+oc_forward_df <-
+  function(placename,
+           key = oc_key(),
+           bounds = NULL,
+           countrycode = NULL,
+           language = NULL,
+           limit = 10,
```

Comment:
Maybe set the default to `limit = 1`. If you geocode multiple addresses in a dataframe you usually want to have one result per address, I think. Said differently, it may be confusing to have multiple results per source address.

Reply:
I am totally fine with using default of 1 result with data frame output.